contracts/two_prisoners_dilemma/.gitignore

```diff
@@ -0,0 +1,9 @@
+# Ignore build artifacts from the local tests sub-crate.
+/target/
```

Comment:
there's no need to have another .gitignore, just specify them in the one in the projects root no? the whole .idea directory looks like something your IDE created, so likely does not belong under version control either?

Reply:
Not sure how this slipped through, I think `cargo contract new` made this .gitignore, but till I'm not sure how I added it to VCS. Anyway, let me remove it.